nebula/ui/components/MZViewBase.qml

```diff
@@ -13,12 +13,13 @@ Item {
    id: root
 
    property string _menuTitle: ""
+   property string accessibleName: ""
    property var _menuOnBackClicked
    property alias _viewContentData: viewContent.data
    property alias _interactive: vpnFlickable.interactive
    property alias _contentHeight: vpnFlickable.contentHeight
 
-   Accessible.name: _menuTitle
+   Accessible.name: (_menuTitle.length > 0) ? _menuTitle : accessibleName
```

Comment:
do you think there is any benefit to doing this over an alias property and leaving the default as `Accessible.name: _menuTitle`

Reply:
Looks like an attached property cannot have an alias property.

Reply:
I always forget about that 🤦